packages/grafana-ui/src/components/Combobox/Combobox.test.tsx

```diff
@@ -132,4 +130,92 @@ describe('Combobox', () => {
       expect(typeof onChangeHandler.mock.calls[1][0].value === 'number').toBeTruthy();
     });
   });
+
+  describe('async', () => {
+    beforeEach(() => {
+      jest.useFakeTimers();
+    });
```

Comment:
we should probably do:
```tsx
afterEach(() => {
  jest.useRealTimers();
});
```

else this will affect the non-async tests as well

Reply:
you can also set up a `user` in this `beforeEach` block, saves you having to remember to put `{delay: null}` in every userEvent call:
```tsx
  let user: ReturnType<typeof userEvent.setup>;

  beforeEach(() => {
    jest.useFakeTimers();
    user = userEvent.setup({ delay: null });
  })
```